exporters/holstebro_helpers.py

```diff
@@ -0,0 +1,811 @@
+                           'Mobile': address['Mobiltelefon'] if 'Mobiltelefon' in address else '',
+                           'Telephone': address['Telefon'] if 'Telefon' in address else '',
+                           'Responsible': f"HK-{responsible}" if responsible != '' else '',
+                           'Company': employee['Org-enhed UUID']
+                           }
+
+                    rows_persons.append(row)
+
+    logger.info(f"Writing {len(rows_persons)} rows to {filename_persons}")
+    mh._write_csv(fieldnames_persons, rows_persons, filename_persons)
+
+    logger.info(f"Writing {len(rows_org)} rows to {filename_org}")
+    mh._write_csv(fieldnames_org, rows_org, filename_org)
+
+
+def update_org_with_hk_managers(mh, nodes):
+    """ Traverses a tree of OUs, for each OU finds the manager of the OU.
+    : param nodes: The nodes of the OU tree
+    """
+    managerHelper = HolstebroHelper(mh)
+
+    for node in PreOrderIter(nodes['root']):
+        ou = mh.read_ou(node.name)
+        # for each ou, check if name contains _leder
+        # if so, check ou for "tilknytninger" and set this as leader for ou.parent
+        if _is_leader_unit(ou):
+            # We have an Afdeling with name _leder, find associated employees and make them leaders in the parent ou
+            associated_employees = mh.read_organisation_people(
+                node.name, 'association', False)
+
+            # Find the manager if an employee is associated with manager departement
+            manager = {}
+            if len(associated_employees) > 0:
+                # Take the first employees associated with the ou and make them leader
+                manager_uuid = list(associated_employees)[0]
+                manager = associated_employees[manager_uuid]
+
+            if len(associated_employees) > 1:
+                # This is an error. There should be no more than one employeen in manager unit.
+                # Log this as an error.
+                logger.error(
+                    "More than one manager associated with: {}".format(ou['name']))
+
+            if ou['parent'] != None:
+                managerHelper.update_manager(
+                    ou['parent'], manager)
+
+            # This manager is now manager for parent ou - or has been removed
+            # if parent ou's name ends with "led-adm", make employee
+            # manager for the parent ou's parent as well.
+            if _is_cm_unit(ou):
+                managerHelper.update_manager(
+                    ou['parent']['parent'], manager)
+
+
+def _is_leader_unit(ou):
+    is_leader_unit = False
+
+    if ou['org_unit_level']['name'] == 'Afdelings-niveau' and ou['name'].count(SETTINGS['imports.holstebro.leaders.manager_extension']) == 1 and ou['name'].count(SETTINGS['imports.holstebro.leaders.manager_prefix_exclude']) == 0:
+        is_leader_unit = True
+
+    return is_leader_unit
```

Comment:
Normalvis siger at linjelængden højest må være 79 tegn (det siger PEP-8 Standarden), nogle bruger 88 tegn.
Denne linje er meget lang.

Jeg ville nok splitte den op:
```suggestion
def _is_leader_unit(ou):
    extensions = SETTINGS["imports.holstebro.leaders.manager_extension"]
    prefix_exclude = SETTINGS["imports.holstebro.leaders.manager_prefix_exclude"]

    num_extensions = ou["name"].count(extensions)
    num_prefix_exclude = ou["name"].count(prefix_exclude)
    if (
        ou["org_unit_level"]["name"] == "Afdelings-niveau"
        and num_extensions == 1
        and num_prefix_exclude == 0
    ):
        return True
    return False
```